docs/openhabian-DEBUG.md

```diff
@@ -6,79 +6,79 @@ source: https://github.com/openhab/openhabian/blob/master/docs/openhabiab-DEBUG.
 
 <!-- Attention authors: Do not edit directly. Please add your changes to the appropriate source repository -->
 
-This document is to give a guiding hand to users when their openHABian install fails in the first place.
+This document is meant to give a guiding hand to users when their openHABian install fails in the first place.
 Read on to find out how to improve debug verbosity and how to proceed with that information.
 
 ## Prerequisites
-First, please make sure you use proper host hardware that is supported as per README.
+First, please make sure you use the proper host hardware that is supported as per [README](https://github.com/blob/master/README.md).
```

Comment:
pls correct the link, https://github.com/openhab/openhabian/blob/master/README.md

Reply:
though the README does not contain the supported HW yet, that's in another open PR..., so we can leave it as it is